cmd/lsif-server/main.go

```diff
@@ -1,45 +1,106 @@
 package main
 
 import (
+	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
+	"path"
 	"strconv"
 	"strings"
 
+	"github.com/sourcegraph/sourcegraph/internal/env"
 	"github.com/sourcegraph/sourcegraph/internal/goreman"
 )
 
+var (
+	servers              = env.Get("LSIF_NUM_SERVERS", "1", "the number of server instances to run (defaults to one)")
+	workers              = env.Get("LSIF_NUM_WORKERS", "1", "the number of worker instances to run (defaults to one)")
+	disablePrometheus, _ = strconv.ParseBool(env.Get("DISABLE_PROMETHEUS", "", "do not run a prometheus process"))
```

Comment:
```suggestion
	disablePrometheus, _ = strconv.ParseBool(env.Get("DISABLE_PROMETHEUS", "false", "do not run a prometheus process"))
```

don't rely on the way `strconv.ParseBool` handles errors (can't parse empty string as bool) to avoid ambiguity

Reply:
Empty string is then false. Isn’t this the idiom we use elsewhere in our codebase? Or is that outdated?

Reply:
Yes, empty string is definitely false but it's relying on error handling behavior which (while definitely defined and reliable) is ambiguous to readers. You can see we almost always specify `true` or `false` instead of an empty string in other locations: https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+strconv.ParseBool%28env.Get&patternType=literal#8